templates/main.php

```diff
@@ -167,46 +167,46 @@ class="collection"
             <div class="app-navigation-entry-menu bubble sortorder-dropdown">
```

Comment:
Sorry, I don't get what you mean. Clicking on the sort-order dropdown button opens the menu, clicking it again, closes it. When I click inside the popover, a sorttype is selected and the menu is closed. What is wrong about that? 😕